Update README.md (#383)
viamrobotics/micro-rdk: examples/esp-idf-component/README.md

# Using Micro-RDK as a library Example

> This example cover shows how to use the Micro-RDK into a new or existing ESP project.

Reviewer: This sentence looks a little off: "cover shows", "use ... into".

Reviewer 2: maybe "for a new or existing..."?

Author: ```suggestion
This example shows how to use the Micro-RDK in a new or existing ESP project.
```
